src/DataWrangling/DataWrangling.jl

```diff
@@ -53,14 +53,24 @@ end
 ##### Downloading utilities
 #####
 
-# netrc-based downloader 
-# This downlader writes down a netrc file with the username and password for the given machine.
-# To avoid storing passwords in plain text, it is recommended to use a temporary directory.
-# For example:
-# mktempdir(parent) do dir
-#     netrc_downloader(username, password, machine, dir)
-#     ... download files ...
-# end
+"""
+    netrc_downloader(username, password, machine, dir)
+
+Create a downloader that uses a netrc file to authenticate with the given machine.
+This downlader writes down a netrc file with the username and password for the given machine in the directory `dir`.
+To avoid storing passwords in plain text, it is recommended to initialize the downloader in a temporary directory.
```

Comment:
How does this avoid storing passwords in plain text?

Reply:
hmmm, well it does but then the temporary directory with the netrc file is deleted when the `do tmp; ...; end` block is closed. So in theory for a finite amount of time the password is indeed stored in the filesystem. I can probably rewrite this comment

Reply:
In theory? I don't understand. The password is _definitely_ stored in plain text right?

Reply:
This is not merely "theoretical", the design explicitly stores it in text.

Reply:
right, it is definitely stored in plain text

Reply:
Also is it a problem that `.netrc` is written? Isn't this the recommended way to use `.netrc` anyways?

Reply:
I think the environment variables route is much cleaner, but it looks like with .netrc we are forced to write down a file. I probably wouldn't like to have a stored file with a password automatically written so I think it is nice to raise this point in the docstring.